DeepSpeech.py

```diff
@@ -573,7 +611,7 @@ def create_inference_graph(batch_size=1, n_steps=16, tflite=False):
 
     if batch_size <= 0:
         # no state management since n_step is expected to be dynamic too (see below)
-        previous_state = previous_state_c = previous_state_h = None
+        previous_states = None
```

Comment:
I don't see where `previous_states` is being used

Reply:
This is a typo from a previous modification that I removed. Nice catch.